### PR TITLE
Fix edit access modal

### PR DESCRIPTION
**Pull request by ZeHiro**
Small regression during lint.

### CODE REVIEW

**Comment by bnjbvr** [comment]
Thanks for the PR! What does it fix, exactly? It seems it's just the exact same code, but reorganized to read differently. What was broken before this PR?

File: client/components/settings/edit-access-modal.js [hunk 1]
```diff
@@ -21,7 +27,9 @@ export default class EditAccessModal extends React.Component {
 
         let customFields;
         if (this.props.customFields) {
-            customFields = this.props.customFields.map(this.extractCustomFieldValue);
+            customFields = this.props.customFields.map(
+                (field, index) => this.extractCustomFieldValue(field, index)
```
Comment (ZeHiro) [inline client/components/settings/edit-access-modal.js]: Here is the real fix. Without this, there is a browser error "this is undefiened" when extractCustomFieldValue is called

Comment (bnjbvr) [inline client/components/settings/edit-access-modal.js]: Haaa I get it. I'd prefer this code to not getting touched, and instead in the ctor that you bind the extractCustomFieldValue to `this` (as for `this.handleSubmit`). Do you mind changing this, please?

Comment (ZeHiro) [inline client/components/settings/edit-access-modal.js]: Done

**Comment by bnjbvr** [comment]
Nice, thanks!

**Comment by bnjbvr** [comment]
Rebased and merged.

**Pull request closed without merging**